mantidproject/mantid
# Resize all widgets in MuonAnalysis when window resized

Adjusted widgets in MuonAnalysis to resize vertically with the window

This focuses on smaller group boxes that would appear squashed on a small screen with high resolution - see the issue. Widgets have been moved into Layouts and given spacers. The aim is that all groupboxes within the window should expand when the window is resized.

**To test:**

Open the interface and drag the window to resize it. Check that all the internal widgets resize too. In particular check the binning option on the Settings tab.

Priority is given to the smaller group boxes (that would be the 'squashed' ones) so you might have to drag it some way before seeing the other boxes expand. If your display has higher resolutions to go to, try these, otherwise you could try using Portrait mode to give the window more room to stretch vertically.

Ideally if you have a small screen with very high resolution, like Aidy's laptop from the issue, that would be the best test of this fix.

Fixes #15739 

[Release notes](https://github.com/mantidproject/mantid/blob/15739_MuonAnalysis_resize_widgets/docs/source/release/v3.7.0/muon.rst#id3) have been updated.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
